lib/ohai/dsl/plugin.rb

```diff
@@ -82,6 +83,10 @@ class Plugin
       include Ohai::Mixin::Command
       include Ohai::Mixin::SecondsToHuman
 
+      # Pending next major release of Ohai.
```

Comment:
Removal pending? The comment is a bit vague. Also when we leave comments in code that say next major version no action is ever taken. Pick the version where we plan to deprecate it.